network/dag/notifier.go

```diff
@@ -291,7 +311,16 @@ func (p *notifier) Notify(event Event) {
 	}
 
 	if err := p.notifyNow(event); err != nil {
-		p.retry(event)
+		retryErrMsg := "Notify event dropped"
```

Comment:
the variable name no longer makes sense.

Reply:
changed to notifyErrMsg